### PR TITLE
RATEST-178:Create  A feature file for Admit Patient.Feature

### PR DESCRIPTION
**Pull request by gracebish**
Issue worked on:
https://issues.openmrs.org/browse/RATEST-178
Changes made:
-Create feature file
-Create steps definitions

### CODE REVIEW

File: src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: Admit to Inpatient
+
+Background:
+    	Given a user clicks on find patient record from home page
+    	When a user selects a patient from the patient list
```
Comment (sherrif10) [inline src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature:5]: Would you allow Given syntax to only be in background stage , As its practically good to have Given scenario in background

**Comment by sherrif10** [comment]
We need to also solve breakages probably you need to first run this test as npm run refappSelenium  to comfirm if your automations are doing the right thing

File: src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature (hunk 1 above)
Comment (gracebish) [reply src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature:5]: Thanks @sherrif10. Let me follow up.

Comment (Irenyak1) [reply src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature:5]: @gracebish this is good. I was going to duplicate your work unknowingly, thank God I thought about looking into this ticket first. Could we have a change of the ticket title from `Create A feature file for Admit Patient.Feature` to `Admit Patient Test workflow`. @sherrif10  and @kdaud  what are your thoughts on this.

**Review by Irenyak1 — COMMENTED** [review]
Since the ticket I am working on builds on this , I do not have to duplicate what you have already done.

**Comment by kdaud** [comment]
RefApp 2.x functionalities that were covered by selenium legacy [tests](https://github.com/openmrs/openmrs-distro-referenceapplication/tree/master/ui-tests/src/test/java/org/openmrs/reference) should not to be written in the new workflows.  See the reasons mentioned on thread public talk post [here](https://talk.openmrs.org/t/missing-feature-and-step-definition-files/34201/5)

Functionalities that were not covered by the selenium legacy test are eligible for the new framework.
cc: @sherrif10 @gracebish @insokwa @Irenyak1

**Comment by sherrif10** [comment]
Hey @kdaud I think we need to clarify these code base well with other
folks, these tests are worth needed reason being this is not covered any
where , however still we need to take caution of tests that are already
written. This will help us not to fall into duplicates. Let me know if am
not clear.

**Comment by kdaud** [comment]
I gave a recommendation on the public talk post [1](https://talk.openmrs.org/t/missing-feature-and-step-definition-files/34201/5) [2](https://talk.openmrs.org/t/missing-feature-and-step-definition-files/34201/6). As well added community member gave a recommendation [here](https://talk.openmrs.org/t/missing-feature-and-step-definition-files/34201/10) !

**Comment by gracebish** [comment]
I am closing th Pr to fix the branch

**Pull request closed without merging**